src/redux/reducers/loginReducer.js

```diff
@@ -0,0 +1,24 @@
+import ActionTypes from '../../constants/ActionTypes'
```

Comment:
one thing I've started to do in other projects is make them more fractal. So instead of `src/views/Login.jsx` and `src/redux/reducers/login.js`, we could just have `src/Login/Login.view.jsx` and `src/Login/reducer.js`. Then we can also co-locate tests, utils, etc alongside the files.

If you agree, I'd like to start moving in that direction.

Shared components can still live in `src/components` or `src/shared` or similar.